pkg/util/wildcard_test.go

```diff
@@ -51,6 +51,24 @@ func TestMatches(t *testing.T) {
 			candidate: "kube-system",
 			matches:   false,
 		},
+		{
+			name:      "wildcard suffix and prefix match",
+			w:         Wildcard("*-kube-*"),
+			candidate: "test-kube-test",
+			matches:   true,
+		},
+		{
+			name:      "no wildcard, only hypens at suffix and prefix",
+			w:         Wildcard("-kube-"),
+			candidate: "test-kube-test",
+			matches:   false,
+		},
+		{
+			name:      "wild card at suffix and prefix, multiple hyphens",
```

Comment:
can we have a double-wildcard and no match, just to make sure that code path gets tested?